### PR TITLE
chore(new-ui): implement new suite title and basics of visual checks mode

### PR DESCRIPTION
**Pull request by shadowusr**
What's done:
- New suite title component, with optimised layout, cleaner look and arrows for quick navigation to next/previous suite
- Now a clean card with a hint shows up when no suite is selected (instead of weirdly empty test info as was before)
- Changed indents in suites tree and steps tree. Now tree leaves are visually nested instead of being at the same level as parent

Test report: https://nda.ya.ru/t/Uru-l5_b78Tzxj

### CODE REVIEW

File: lib/static/new-ui/features/suites/components/SuitesTreeView/selectors.ts [hunk 1]
```diff
@@ -81,28 +91,31 @@ export const getTreeViewItems = createSelector(
...
-                        .filter(Boolean) as TreeViewItem<TreeViewSuiteData | TreeViewBrowserData>[]
-                };
+            const children: TreeViewItem<TreeViewSuiteData | TreeViewBrowserData>[] = [];
+            if (hasBrowsers(suiteData)) {
+                children.push(...suiteData.browserIds
+                    .map((browserId) => formatBrowser(browsers[browserId], data))
+                    .filter(Boolean) as TreeViewItem<TreeViewBrowserData>[]);
+            }
+            if (hasSuites(suiteData)) {
+                children.push(...suiteData.suiteIds
```
Comment (sipayRT) [inline lib/static/new-ui/features/suites/components/SuitesTreeView/selectors.ts:101]: don't use spread here, pls. this operation is quite slow on large arrays

File: lib/static/new-ui/features/visual-checks/components/VisualChecksPage/index.module.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.card {
+    height: calc(100vh - 20px);
+}
+
+.card__title {
+    margin-bottom: 8px !important;
```
Comment (sipayRT) [inline lib/static/new-ui/features/visual-checks/components/VisualChecksPage/index.module.css:6]: why are you using `!important` here? isn't it enough just to increase the weight of the selector by nesting them (`.card .cart__title`)?

File: lib/static/new-ui/types/store.ts [hunk 3]
```diff
@@ -41,6 +42,9 @@ export interface ResultEntityCommon {
     suiteUrl?: string;
     history?: TestStepCompressed[];
     error?: TestError;
+    suitePath: string[];
+    /** @node Browser Name/ID, e.g. `chrome-desktop` */
```
Comment (sipayRT) [inline lib/static/new-ui/types/store.ts:46]: did you mean `note`?

**Review by sipayRT — APPROVED** [review]

File: lib/static/new-ui/components/Card/UiCard.module.css [hunk 4]
```diff
@@ -0,0 +1,8 @@
+.card {
```
[the remark this answers is not in the document]
Comment (shadowusr) [reply lib/static/new-ui/components/Card/UiCard.module.css:1]: Did exactly that in one of the PRs on top of this one!

File: lib/static/new-ui/components/Card/index.module.css [hunk 5]
```diff
@@ -3,10 +3,10 @@
     position: relative;
 }
 
-.card {
-    box-shadow: rgb(255, 255, 255) 0 0 0 0, rgba(9, 9, 11, 0.05) 0 0 0 1px, rgba(0, 0, 0, 0.05) 0 1px 2px 0;
-}
+.card {}
```
[the remark this answers is not in the document]
Comment (shadowusr) [reply lib/static/new-ui/components/Card/index.module.css:6]: Got rid of this empty class.

File: lib/static/modules/reducers/visual-checks-page.ts [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import {State} from '@/static/new-ui/types/store';
+import actionNames from '@/static/modules/action-names';
+import {applyStateUpdate} from '@/static/modules/utils/state';
+import {VisualChecksPageAction} from '@/static/modules/actions';
+
+export default (state: State, action: VisualChecksPageAction): State => {
+    switch (action.type) {
+        case actionNames.VISUAL_CHECKS_PAGE_SET_CURRENT_NAMED_IMAGE:
```
[the remark this answers is not in the document]
Comment (shadowusr) [reply lib/static/modules/reducers/visual-checks-page.ts:8]: I agree; But this issue will be resolved once we migrate to RTK/store slices, that will be automatically namespaced. For now I think `VCP` is not very obvious, so I decided to keep it as is

File: lib/static/new-ui/components/SuiteTitle/index.tsx [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+    return <div className={classNames(styles.container, props.className)}>
+        <div className={styles.content}>
+            <div className={classNames(styles.breadcrumbs)}>
+                {suitePath.map((item, index) => (
+                    <div key={index} className={styles.breadcrumbsItem}>
+                        {item}
+                        {index !== suitePath.length - 1 &&
+                            <div className={styles.separator}>
+                                <ChevronRight height={11}/>
```
[the remark this answers is not in the document]
Comment (shadowusr) [reply lib/static/new-ui/components/SuiteTitle/index.tsx:34]: changed to 12

**Pull request merged**